modules/osdk-cli-reference-alpha.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-cli-reference.adoc
+
+[id="osdk-cli-reference-alpha_{context}"]
+= alpha
+
+The `operator-sdk alpha` command is used to run an alpha subcommand.
```

Comment:
If you have access to an SME, can you check if `alpha` corresponds to the status of the command? I see in 4.9 that scorecard is a top-level command.  I'm speculating.

Reply:
4.7 onwards it's top level, in 4.6 it comes under alpha. i.e. we cannot have a command `operator-sdk scorecard` it has to be `operator-sdk alpha scorecard` where `scorecard` is a subcommand.